src/Deployment/source.extension.vsixmanifest

```diff
@@ -10,41 +10,41 @@
   </Installation>
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-    
-    <Dependency d:ProjectName="CompilerExtension" 
-                DisplayName="Roslyn Compilers" 
-                Version="[|CompilerExtension;GetBuildVersion|,)"
-                d:Source="Project" 
-                d:InstallSource="Embed" 
-                d:VsixSubPath="Vsixes" 
-                Location="|CompilerExtension;VSIXContainerProjectOutputGroup|" 
+
+    <Dependency d:ProjectName="CompilerExtension"
+                DisplayName="Roslyn Compilers"
+                Version="[|%CurrentProject%;GetBuildVersion|,)"
```

Comment:
Can you explain why this makes a difference?

Reply:
My guess is the manifest handling doesn't properly account for VSIX sub path when translating this syntax.